### DISCUSSION TITLE
Image export

### DISCUSSION THREAD

**Discussion by gfghy** [Open-Systems-Pharmacology/Forum#971] [How-Tos] [answered]
How to export the image of prediction lines and observations in high resolution?
I found that I could only export prediction lines without observation points. Hope for your answer, thank you very much!

**Comment by StephanSchaller** [comment]
Hi @gfghy ,

is this what you are looking for:
![image](https://user-images.githubusercontent.com/25477643/134640748-b838427d-c257-4435-bd64-cdd8fcee1e19.png)

This will export, both simulated curves and observed data point in a figure.
Otherwise right-click on your figure --> export to excel and plot it in excel or with R etc.

**Reply by gfghy** [reply]
Thank you for your answer. However, I still have several questions: 1. After exporting the data points of the prediction line to Excel, how to make the prediction line fit well with the measured points? 2. How to add confidence intervals? 
Hope for your further answers.
Best regards.

**Comment by StephanSchaller** [comment]
Hi @gfghy , for this we have parameter identification directly in PK-Sim: https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification

**Reply by gfghy** [reply]
Thank you for your answer again. But i mean that i do not know how to add the confidence intervals after drawing the prediction lines and observations outside the software? And back to the first question, can you tell me how to draw the prediction line and make it fit well with the measured points after export "Time Profile Analysis" to excel? Because the image i export only contain a predicted file like the following figure, i have to use the exported data of predicted file and observed data to make a piture with both of them. But i am not good at it. I really need your help. If you have any other efficient way, please tell me, thank you very much.
Looking forward to your answer.
Best wishes.

**Reply by StephanSchaller** [reply]
> Because the image i export only contain a predicted file like the following figure

I can't see a figure. Please copy&paste into the chat. Thx

**Comment by gfghy** [comment]
Sorry, the pictures are here.
[plot0.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/7225800/plot0.pdf)
![微信图片_20210924191649](https://user-images.githubusercontent.com/91039764/134679498-a88bd076-f5b3-4fcf-80fb-327488a95449.jpg)

**Comment by msevestre** [comment, marked as answer]
@gfghy As @StephanSchaller suggested:
If you want to export the graph as is, use Export to clipboard
![image](https://user-images.githubusercontent.com/1041237/134683793-db922bbd-ea12-404e-bfee-5e045ee53d6c.png)


If you want to export the data, use export to excel
This will export one sheet for the simulated values and one sheet per observed data
![image](https://user-images.githubusercontent.com/1041237/134684070-936a4418-92f4-473c-a2d6-f35c62f109e3.png)

Once the data is exported to Excel, you can use any plot capabilities from Excel and draw lines, dots etc... But how to do this in Excel is outside of the scope of this Forum.

**Reply by msevestre** [reply]
and I should add: Export to pdf is broken at the moment and does not export obs data associated to a plot.

**Reply by msevestre** [reply]
Simply disable the watermark option in your settings
![image](https://user-images.githubusercontent.com/1041237/134690669-519016cd-577d-4eec-9963-40f93f30ea60.png)

**Reply by prvmalik** [reply]
You have to go into options > remove watermark.

Also click on chart options > chart color > white to make it look better. good luck.

**Reply by gfghy** [reply]
Thank you, do I have to use Mobi for this? But the picture exported this way is lack of clarity.

And maybe i can know when will the export image function be fixed? Thanks a lot.

**Reply by msevestre** [reply]
>Thank you, do I have to use Mobi for this? But the picture exported this way is lack of clarity.

No this works in PKSim as well. And as mentioned above, you can specify the size of the picture to export in the export tab. 

>And maybe i can know when will the export image function be fixed? Thanks a lot.

Not in the near future. I cannot give you a time frame at the moment

**Reply by gfghy** [reply]
Thanks a lot! But I do hope you can fix it soon, cause it is more efficient.  Thank you again, lovely people.

**Reply by StephanSchaller** [reply]
I showed you above, where you can set the resolution for your export figure

![image](https://user-images.githubusercontent.com/25477643/134762890-1c20c4bf-cc2e-4e38-95c0-984313f49349.png)
